### PR TITLE
Add rule generation from regex

### PR DESCRIPTION
**Pull request by DoomCoder**
Creates alternative lemma_sources/_*rules.py files, places them in lemma_sources_exp folder

### CODE REVIEW

File: lemma_rules_extraction/yield_all_suffixes.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    expanded_rules = []
+    for rule in rules:
+        replacement = rule[1]
+        regex = rule[0]
+        suffixes = list(sre_yield.AllStrings(rule[0], charset=charset))
+        expanded_rules += [(suf, regex, replacement) for suf in suffixes]
+
+    return expanded_rules
+
+if __name__ == "__main__":
```
Comment (kowaalczyk) [inline lemma_rules_extraction/yield_all_suffixes.py]: Use argparse or click plz, with arguments for output and prefix at least.

File: tests/lemmatizer_sanity_check.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 from evaluation.lemmatizer_predict import load_and_predict_lemmas
 
 
-if __name__ = '__main__':
+if __name__ == '__main__':
```
Comment (kowaalczyk) [inline tests/lemmatizer_sanity_check.py]: Argparse or click, same as in other scripts please

File: words.json.dvc [hunk 3]
```diff
@@ -1,9 +1,12 @@
 cmd: python ./lemma_rules_extraction/assign_lemmas.py
 deps:
```
Comment (kowaalczyk) [inline words.json.dvc]: Python file not in deps

File: rules.json.dvc [hunk 4]
```diff
@@ -4,8 +4,11 @@ deps:
   path: data/lemmatizer_data/polish.aff
```
Comment (kowaalczyk) [inline rules.json.dvc]: Python file not in deps

File: polish.dic.dvc [hunk 5]
```diff
@@ -1,12 +1,19 @@
 cmd: ' python ./lemma_rules_extraction/get_ispell.py'
```
Comment (kowaalczyk) [inline polish.dic.dvc]: Add python file to deps

File: lemma_sources.dvc [hunk 6]
```diff
@@ -2,12 +2,15 @@ cmd: ' python ./lemma_rules_extraction/prepare_sources.py'
 deps:
```
Comment (kowaalczyk) [inline lemma_sources.dvc]: Add python file to deps

**Comment by DoomCoder** [comment]
@kowaalczyk Changes applied, please tell me if it's ok to leave expanded rules dvc file unchanged (using default output directory)

**Pull request merged**